crates/astria-core/src/protocol/genesis/v1alpha1.rs

```diff
@@ -542,80 +559,183 @@ impl From<raw::IbcParameters> for IBCParameters {
 }
 
 #[derive(Clone, Debug)]
-pub struct Fees {
-    pub transfer_base_fee: u128,
-    pub sequence_base_fee: u128,
-    pub sequence_byte_cost_multiplier: u128,
-    pub init_bridge_account_base_fee: u128,
-    pub bridge_lock_byte_cost_multiplier: u128,
-    pub bridge_sudo_change_fee: u128,
-    pub ics20_withdrawal_base_fee: u128,
+pub struct GenesisFees {
+    pub sequence: SequenceFeeComponents,
+    pub transfer: TransferFeeComponents,
+    pub ics20_withdrawal: Ics20WithdrawalFeeComponents,
+    pub init_bridge_account: InitBridgeAccountFeeComponents,
+    pub bridge_lock: BridgeLockFeeComponents,
+    pub bridge_unlock: BridgeUnlockFeeComponents,
+    pub bridge_sudo_change: BridgeSudoChangeFeeComponents,
+    pub ibc_relay: IbcRelayFeeComponents,
+    pub validator_update: ValidatorUpdateFeeComponents,
+    pub fee_asset_change: FeeAssetChangeFeeComponents,
+    pub fee_change: FeeChangeFeeComponents,
+    pub ibc_relayer_change: IbcRelayerChangeFeeComponents,
+    pub sudo_address_change: SudoAddressChangeFeeComponents,
+    pub ibc_sudo_change: IbcSudoChangeFeeComponents,
```

Comment:
Should most of these be `Option`? It seems the only which stricly must be set is `fee_change`?

Reply:
Tracking issue: https://github.com/astriaorg/astria/issues/1662